README.md

```diff
@@ -1,4 +1,27 @@
 # What is ActiveRecord ?
 
-1. What is ActiveRecord? 
-2. ActiveRecord vs. SQL
\ No newline at end of file
+## Objectives
+
+1. Define ActiveRecord and understand why it is easier to use.
+2. Compare ActiveRecord vs. SQL queries.
+
+## ActiveRecord
+
+ActiveRecord is what is referred to as an Object-Relational Mapper (ORM). It's responsible for mapping exactly one row in a database to one object in Ruby. 
+
+What this means in practice is that you use ActiveRecord to create or retrieve your data and use traditional Ruby methods and attribute assignments to interact with this data. Instead of composing an SQL query manually, filling in the search parameters, sending it off to a database API, running the query, examining and parsing the result and storing the result in either a hash or a database, you let ActiveRecord do all that for you.
+
+ActiveRecord is also responsible for handling all database interaction from the application level.
+
+
+## ActiveRecord vs SQL 
+
+| SQL                                   | ActiveRecord          |
+|----					                       |-----                  |
+|SELECT * FROM cats;                    | Cat.all              |  
+|SELECT name FROM cats; 		           | Cat.all.name         | 
+|SELECT * FROM cats WHERE name = "Maru";| Cat.where(name: 'Maru')|
+|SELECT * FROM cats WHERE age > 2;      | Cat.where('age > 2')|
+|UPDATE cats SET name = "Hana" WHERE name = "Hannah";| hannah = Cat.where(name: 'Hana') <br> hannah.name = "Hannah" |
+|DELETE * FROM cats WHERE id = 3        | Cat.destroy(3)|
+
```

Comment:
Maybe a follow up: "Notice how much cleaner and more straightforward ActiveRecord makes interacting with the database - you'll learn more about CRUD methods in ActiveRecord later in this unit."